src/EventStore/EventStore.spec.ts

```diff
@@ -81,14 +81,14 @@ describe('EventStore', () => {
 
   describe('.getItem', () => {
     it('should return null for invalid key', async () => {
-      Persistence.getItem('not-a-real-key')
+      Persistence.LocalStore.getItem('not-a-real-key')
```

Comment:
@OR13 should probably throw an invalid key error or the like and catch it here instead of returning a null model.